docs/vereisten/index.md

```diff
@@ -1,69 +1,88 @@
-
-!!! warning "Opmerking"
-    Het algoritmekader is nog volop in ontwikkeling. 
-    Dit betekent dat de lijst met vereisten die nu zijn opgenomen in het algoritmekader nog onvolledig is, en dat de vereisten later nog kunnen worden aangepast. 
-    Meer vereisten volgen in een volgende versie van het Algoritmekader. 
-
-## Welke vereisten zijn wanneer van toepassing?
-In dit deel van het Algoritmekader kan worden genavigeerd door de verschillende vereisten. 
-Wat precies van toepassing is, is sterk afhankelijk van de specifieke toepassing. 
-Daarom worden de vereisten gekoppeld aan het type technologie (rekenregel, machine learning of generatieve AI) en aan de risico classificatie (niet-impactvol, impactvol en hoge risico). 
-Dit moet gebruikers helpen om te duiden welke vereisten in welke situaties moeten worden nageleefd. 
-
-??? note "Opmerking"
-    Aan deze functionaliteit wordt nog gewerkt. Meer hierover zal volgen in een volgende versie. 
+## Aantal eisen verschilt per situatie
+Welke eisen gelden voor jouw organisatie, hangt af van:
+- de technologie die je gebruikt: rekenregels, machinelearning of generatieve AI
+- het risiconiveau van je product
+- de [fase in de levenscyclus](https://minbzk.github.io/Algoritmekader/levenscyclus/) van je project
 
 ### Voorbeeld
-Ter illustratie, ongeacht het type technologie, zijn discriminerende algoritmes en AI verboden en het vereiste van non-discriminatie moet in alle gevallen worden toegepast. 
-De mate waarin dit speelt kan verschillen en het aantal te treffen maatregelen ook. 
-Organisaties zullen hier hoe dan ook aandacht aan moeten besteden. 
-Dit is anders als het gaat om bijvoorbeeld het vereiste om algoritmes en AI op te nemen in het Algoritmeregister. 
-Dit is van belang bij impactvolle algoritmische toepassingen en hoog-risico AI-systemen. 
-Voor de niet-impactvolle rekenregels is dit niet noodzakelijk en kan daardoor achterwege blijven. 
-
-## Overzicht van vereisten
-
-Onderstaand volgt een overzicht van vereisten per fase van de levenscyclus. 
-
-??? simple "Probleemanalyse"
-
-    <!-- list_vereisten_1 levenscyclus/probleemanalyse -->
-
-??? simple "Ontwerp"
-
-    <!-- list_vereisten_1 levenscyclus/ontwerp -->
-
-??? simple "Dataverkenning en datapreparatie"
-
-    <!-- list_vereisten_1 levenscyclus/dataverkenning-en-datapreparatie -->
-
-??? simple "Ontwikkelen"
-
-    <!-- list_vereisten_1 levenscyclus/ontwikkelen -->
-
-??? simple "Verificatie en validatie"
-
-    <!-- list_vereisten_1 levenscyclus/verificatie-en-validatie -->
-    
-??? simple "Implementatie"
-
-    <!-- list_vereisten_1 levenscyclus/implementatie -->
-    
-??? simple "Monitoring en beheer"
-
-    <!-- list_vereisten_1 levenscyclus/monitoring-en-beheer -->
-    
-??? simple "Uitfaseren"
-
-    <!-- list_vereisten_1 levenscyclus/uitfaseren -->
+> Impactvolle algoritmes en AI-systemen worden gepubliceerd in het Nederlandse algoritmeregister.
```

Comment:
Ik twijfel of dit nou het beste voorbeeld is? Misschien iets gebruiken uit de AI-verordening? Daar zijn veel vereisten alleen van toepassing als je onder hoog-risico valt. Als je niet hoog-risico bent, is die vereiste dus niet van toepassing.

Aan de andere kant geldt bijvoorbeeld non-discriminatie wetgeving altijd, en dus ook voor alle algoritmes

Reply:
Dit voorbeeld stond in de huidige tekst. Ik kan zelf niet beoordelen wat een beter voorbeeld is, daarom laat ik het voor nu maar staan.